Index.html

```diff
@@ -0,0 +1,111 @@
+                    <img src="green-icon.png" href="quickbooks" id="bottom-logo">Quickbooks
+                    <img src="red-icon.png" href="TurboTax" id="bottom-logo">TurboTax
+                </div>
+        </figure>
+    </section>
+
+    <fieldset class="full-form">
+        <form class="form-elements">
+            <div class="formHeading">
+                <h2>Create an Intuit account</h2>
+            </div>
+            <div class="subHeading">
+                <small>One account for everything Intuit, including Mint.</small>
+            </br>
+                <small style="font-size: smaller; color: blue;">learn more</small>
+            </br>
+                <small>Already have an account?</small><small style="color: blue;"> Sign In</small>
+            </div>
+            </br>
+            <div class="inputs" >
+                <div class="email">
+                    <label for="email-input">Email address</label>
+                    <input type="email" id="email-input" name="email" required>
+                </div>
+                <div class="phone">
+                    <label for="phone-input">Phone (recommended)</label>                                    
+                    <input type="tel" id="phone-input" name="phone" role="alert"> 
+        
+                </div>
+               
+                <div class="password" >
+                    <label for="password-input">Password</label>
+                    <input type="password" id="password-input" name="password" required>
+                </div>  
+                <div class="subBtn">
+                    <button type="submit"><i class="fas fa-lock"></i> Create Account</button>  
+                </div>
+            </div>
+            
+            <div class="sign-up-footer">
+                <p class="agreements">By clicking Create account, you agree to our<br/>
+                    <a href="#terms">Terms</a> and have read and acknowledge our 
+                    <a href="#privacy">Privacy<br/>Statement</a>.
+                </p>
+            </div>
+    
+            <div class="grey-background">
+                <p class="captcha">Invisible reCAPTCHA by Google 
+                    <a href="#privacy-policy">Privacy Policy</a> and
+                    <a href="#terms">Terms of Use</a>.
+                </p>
+            </div>
+            
+ 
+        </form>
+    </fieldset>
+  
+</body>
+
+<footer class="footer-element">
+    <ul class="ul-footer">
+        <li>Legal</li>
+        <li>Privacy</li>
+        <li>Security</li>
+    </ul>
+    <p class="footer-paragraph">
+        Legal Privacy Security
+        © 2020 Intuit, Inc. All rights reserved. Intuit, 
+        QuickBooks, QB, TurboTax, ProConnect and Mint are registered trademarks of Intuit Inc.<br/>
+        Terms and conditions, features, support, pricing, and service options 
+        subject to change without notice.</p>
+</footer>
```

Comment:
- [ ] The footer must be inside the body tags. Kindly move it at the end of the body tags.